release-notes/opensearch-dashboards-search-relevance.release-notes-2.7.0.0.md

```diff
@@ -0,0 +1,17 @@
+## What's Changed
+* **Welcome** Sean Li and Louis Chu as maintainers on the repo by @macohen in https://github.com/opensearch-project/dashboards-search-relevance/pull/172
+
+## Measurement
+* Instrument metrics on server-side by @noCharger in https://github.com/opensearch-project/dashboards-search-relevance/pull/162
+* Refactor metric names by @noCharger in https://github.com/opensearch-project/dashboards-search-relevance/pull/174
+* [Bug][Build] export config file by @kavilla in https://github.com/opensearch-project/dashboards-search-relevance/pull/183
+
+## Maintenance 
+* Renamed and moved codecov.yml to root for coverage enforcement by @macohen in https://github.com/opensearch-project/dashboards-search-relevance/pull/161
+* Run Cypress integration tests on build by @msfroh in https://github.com/opensearch-project/dashboards-search-relevance/pull/143
```

Comment:
@msfroh is this one ready for backport 2.7?

Reply:
going to remove this one since it isn't being backported or 2.7. can we do this for 2.8?